src/unity/toolkits/tcmps/CMakeLists.txt

```diff
@@ -1,6 +1,34 @@
 project(unity_toolkits)
 
 if(APPLE AND HAS_MPS AND NOT TC_BUILD_IOS)
+    # We need to get the proper SDK paths in the build
+    execute_process(
```

Comment:
Same comment; duplicated with logic above. Let's see if we can get a single top level CMAKE_OSX_SYSROOT to result in the correct paths for everything else. The `find_library` calls are probably still needed, but using default paths should do the right thing now that that variable is set.